assets/js/state/selectors/activityLogsSettings.js

```diff
@@ -0,0 +1,36 @@
+import { createSelector } from '@reduxjs/toolkit';
+
+export const getActivityLogsSettings = createSelector(
+  [({ activityLogsSettings }) => activityLogsSettings],
+  ({
+    settings,
+    errors,
+    loading,
+    editing,
+    networkError,
+    testingConnection,
+  }) => ({
+    settings,
+    errors,
+    loading,
+    editing,
+    networkError,
+    testingConnection,
+  })
+);
+
+export const getActivityLogsSettingsLoading = createSelector(
```

Comment:
I could not find usages of `getActivityLogsSettingsLoading` except of its definition and test.

Maybe I am missing some intent to use it somewhere, however if not needed we could remove it.